_posts/2022-06-26-contextual-slu.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Incorporating context to improve SLU
+date: 2022-06-19
+tags: [slu, context, nlp]
+categories: [Machine Learning]
+# image: assets/images/label-noise-blog/label-noise.png
+layout: post
+authors: [sanchit-ahuja]
+latex: True
+---
+
+## Introduction
+In task-oriented dialogue systems, the spoken language understanding, or SLU, refers to the task of parsing the natural language utterances into
+semantic frames. The problem of contextual SLU majorly focuses on effectively incorporating dialogue information.
+
+
+## Why context?
+There are many in-scope intents that contain a lot of contextual information within the bot prompt and/or by using State-level information. Examples from one of our clients are below.
+|                                                                                 **Bot Prompt**                                                                                | **User Response** |        **Intent**        |
+|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:-----------------:|:------------------------:|
+| Hi! I am Divya, your Hathway virtual assistant. Are you looking for a new Hathway Broadband connection?                                                                       | yes** **          | confirm_new_connection   |
+| Alright! To check your internet connectivity status, the device should be connected to the power supply and switched on. Will it be possible for you to switch on the device? | yes               | confirm                  |
+| Okay!. To start with, please tell me if you are next to your Hathway device?                                                                                                  | I am              | confirm_near_device** ** |
+
+
+Having that extra context from the bot can definitely help us to make a better contextual model.
```

Comment:
In the above example, all the intents seem utterance based. Can you instead of this example, add a different one, where context helps disambiguate or provide critical information needed to infer the intent.